### PR TITLE
DR-1297 Provide step class names for logging; quiet down queue logging

### PR DESCRIPTION
**Pull request by ddietterich**
There are three changes:
1. Set the scheduling and queue logging to debug. get rid of that every five second blat in our data repo logs
2. Build a list of step class names as the flight is being constructed. pass those names over to the flight context so they are available to the hooks.
3. Opportunistically add `final` where IntelliJ tells me to

### CODE REVIEW

File: src/main/java/bio/terra/stairway/WorkQueueListener.java [hunk 1]
```diff
@@ -24,22 +25,18 @@ public WorkQueueListener(Stairway stairway, Queue workQueue) {
...
+  // happens we sleep for NO_PULL_SLEEP_SECONDS and then check the queue depth.
   //
   // The reason to limit to MAX_MESSAGES_PER_PULL is that the evaluation of queue depth is done at,
-  // say,
-  // time T0, but the pull will wait for message arrival from pubsub at, say, time T1. By time T1,
-  // the
-  // queue depth situation might have changed. So we don't want to set up to grab lots of messages
-  // and
-  // then find we are queuing flights way over maxQueuedFlights.
+  // say,time T0, but the pull will wait for message arrival from pubsub at, say, time T1.
```
Comment (fboulnois) [inline src/main/java/bio/terra/stairway/WorkQueueListener.java]:
```suggestion
  // say, time T0, but the pull will wait for message arrival from pubsub at, say, time T1.
```

**Review by snf2ye — APPROVED** [review]
Thank you - I think these logs will be helpful!

**Review by fboulnois — APPROVED** [review]
lgtm, micro-nit 🦠 :

File: src/main/java/bio/terra/stairway/WorkQueueListener.java (hunk 1 above)
Comment (ddietterich) [reply src/main/java/bio/terra/stairway/WorkQueueListener.java]: Thanks for that!

**Pull request merged**